### PR TITLE
Edits for Cert-Manager README

### PR DESCRIPTION
**Pull request by yGabriel**
## BACKGROUND
* Add information about the background and reason for the change.

-----------------------------------------------------------------------

## Changes
* List the changes that you made

-----------------------------------------------------------------------

## Checklist
- [ ] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

File: stacks/cert-manager/README.md [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
-
-`Cert-Manager` goes hand in hand with your `Ingress Controller` resource. An `Ingress Controller` is the main entrypoint of your `Kubernetes` cluster, and sits in front of your `backend services`. So, by securing your ingress resources (e.g. `Nginx`), sensitive data never goes in or out un-encrypted. On top of that, you provide identity information to users, by presenting them a valid SSL/TLS certificate whenever they visit your website(s). Self-signed certificates are not to be trusted, and should never be used in production systems.
-
-Please make sure to visit the Cert-Manager [official documentation](https://cert-manager.io/docs/) page to study more.
+See Cert-Manager's [official documentation](https://cert-manager.io/docs/).
 
 **Note:**
 
-DigitalOcean is using `Helm v3` to deploy `Cert-Manager` to your `DOKS` cluster.
+DigitalOcean is using Helm v3 to deploy Cert-Manager to your DOKS cluster.
```
Comment (caquillo07) [inline stacks/cert-manager/README.md:11]: question: In every other PR these are hyper linked elsewhere the first time they show up, and they are also marked as `code` every where here in the PR. 

Are these code tags meant to be removed? Just want to make sure this was intended :)

**Review by caquillo07 — APPROVED** [review]

File: stacks/cert-manager/README.md (hunk 1 above)
Comment (yGabriel) [reply stacks/cert-manager/README.md:11]: Yes, it was intended! Since these terms aren't code themselves, they shouldn't be in code spans.

Comment (sharmita3) [reply stacks/cert-manager/README.md:11]: @caquillo07 This is consistent with the other edits: https://github.com/digitalocean/marketplace-kubernetes/pull/344/files#diff-09d0bac7f02052be60b7f3b55ecc4ec9866ecb6084c5e21ee41df3c51cb55d55R27 and https://github.com/digitalocean/marketplace-kubernetes/pull/330/files#diff-8a86b5d2ad6f5bd03c401797fb91fe180a96d560d8550deeaa2989fc9531644aR15.

Comment (caquillo07) [reply stacks/cert-manager/README.md:11]: sounds good!

**Pull request merged**